rewrite-java/src/main/java/org/openrewrite/java/style/IntelliJ.java

```diff
@@ -43,7 +43,7 @@ public static IntelliJ defaults() {
 
     @JsonIgnore
     @Override
-    public Collection<Style> getStyles() {
+    public Collection<? extends Style> getStyles() {
```

Comment:
I wouldn't widen this.

Reply:
Reverted changes made against `NamedStyle` and changes up the tree for the moment-- when you say wouldn't widen this, is that specific to this `getStyles` override, or to the widening of `Collection<Styles>` as part of `DeclarativeNamedStyles` + `NamedStyle` all the way up?